packages/clay-form/stories/index.tsx

```diff
@@ -87,11 +87,15 @@ storiesOf('Components|ClayDualListBox', module).add('default', () => {
 		<ClayDualListBox
 			disableLTR={
 				boolean('disableLTR', false) ||
-				secondSelectBoxItems.length >= rightMaxItems
+				secondSelectBoxItems.length >= rightMaxItems ||
+				!firstSelectBoxItems.length ||
+				!leftSelected.length
```

Comment:
I don't think we shouldn't put this responsibility on the user to manage these checks, I think we should be able to internalize.

Reply:
Yea, that makes sense, I'll get on top of that!